src/index.html

```diff
@@ -11,6 +11,57 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="get"
+    data-qa="big"
+    class="search-form"
```

Comment:
Attributes should have 2 spaces indent if moved to a new line
```suggestion
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="get"
      data-qa="big"
      class="search-form"
```